src/rdkb/impl.c

```diff
@@ -53,6 +53,8 @@
 #define DEVICE_MAC                   "Device.DeviceInfo.X_COMCAST-COM_WAN_MAC"
 #elif defined(_HUB4_PRODUCT_REQ_)
 #define DEVICE_MAC                   "Device.DeviceInfo.X_COMCAST-COM_WAN_MAC"
+#elif defined(RDKB_EXTENDER)
```

Comment:
Is this RDKB_EXTENDER not used in any other platforms like POD etc.

Reply:
Changed back to XLE device-specific flag, instead of feature specific, as RDKB_EXTENDER flag may be used on other device as well.